docs/Changelog.md

```diff
@@ -20,10 +20,13 @@
 * [#4101](https://github.com/TouK/nussknacker/pull/4101) Notifications fixes:
   * Scenario state wasn't refreshed after deploy/cancel action was successfully finished (was only after failure)
   * Notification "Deployment of ... failed ..." was presented even for cancel action 
-* [#4100](https://github.com/TouK/nussknacker/pull/4100) "In progress" state is now presented not only inside scenario editor but also on scenarios list
 * [#4102](https://github.com/TouK/nussknacker/pull/4102) Flink deploy now wait until job is started on TaskManagers before reporting that is finished - 
   thanks to that status and versions panel are refreshed with "DEPLOYED" state in the same time
 * [#3992](https://github.com/TouK/nussknacker/pull/3992) Fix for compiling scenarios containing filter node with only 'false' edge
+* [#4100](https://github.com/TouK/nussknacker/pull/4100)[#4104](https://github.com/TouK/nussknacker/pull/4104)
+  Before the change, in scenario list, were presented "local" scenario states - based only on Nussknacker's actions log.
```

Comment:
Maybe rephrase to?
> Before the change, the scenario list presented "local" states - based only on Nussknacker's actions log.

Reply:
It looks much simpler and better. Thanks!